backend/README.md

```diff
@@ -83,35 +89,71 @@ Example query located in`database/examples/basic_search_query.json`:
 		"results_first_posted_date_to":"",
 		"last_update_posted_date_from":"",
 		"last_update_posted_date_to":"",
-	    "first":"",
-	    "last":""
+		"study_results":"",
+		"study_type":"",
+		"eligibility_age":"",
+		"eligibility_min_age":"25",
+		"eligibility_max_age":"75",
+		"eligibility_gender":"Female",
+		"eligibility_ethnicity":"",
+		"eligibility_condition":"",
+		"eligibility_healthy_volunteer":"",
+		"study_title_acronym":"",
+		"study_outcome_measure":"",
+		"study_collaborator":"",
+		"study_ids":"",
+		"study_location_terms":"",
+		"study_funder_type":"Industry",
+		"study_document_type":"Study Protocol",
+		"study_results_submitted":"",
+		"study_roa":"",
+		"first":"",
+		"last":"",
+		"metadata_required":"True"
 	}
 
+
+1. Pass all of the above 'post' parameters with exact name. You can choose to pass only those parameters that have value entered/selected by user. You don't need to pass all parameters. If none of the parameters are passed, the API will get first 100 records from database.
+
+1. 'first' and 'last' parameters are for pagination. If these parameters are blank or not passed, Django API will assume first = 0 and last = 100.
+
 Example curl command with a timer using the example json:
 
-`time curl -X POST http://127.0.0.1:8000/ctgov/api/basic_search -d database/examples/basic_search_query.json`
+`time curl -X POST http://127.0.0.1:8000/ctgov/api/search_studies -d database/examples/basic_search_query.json`
 
 #### Output
 
+<span style="color:red">Please note search results now contains "metadata" information. This means ReactJS parsing needs to be modified accordingly.</span>
```

Comment:
Remove span and style. The color is not showing up. I don't think it is possible to color plain text in Github markdown.

Reply:
I can remove it.